### PR TITLE
Fallbacks: flip anchor functions that are nested in other function calls

### PR DESCRIPTION
**Pull request by tuespetre**
Not aware of an existing open issue for this, but I encountered it today.

### CODE REVIEW

File: src/fallback.ts [hunk 1]
```diff
@@ -421,15 +420,19 @@ export function applyTryTacticToBlock(
       declarations[key] ??= 'revert';
     }
 
-    // todo: This does not support anchor functions that are nested or passed
-    // through custom properties.
-    if (isAnchorFunction(valueAst.children.first)) {
-      valueAst.children.first.children.forEach((item) => {
-        if (isIdentifier(item) && isAnchorSide(item.name)) {
-          item.name = mapAnchorSide(item.name, tactic);
+    // todo: This does not support anchor functions that are passed through custom properties.
```
Comment (jgerigmeyer) [inline src/fallback.ts:423]:
```suggestion
    // todo: This does not support percentage anchor-side values, nor anchor
    // functions that are passed through custom properties.
```

**Review by jgerigmeyer — APPROVED** [review]
Thanks!

File: src/fallback.ts [hunk 2]
```diff
@@ -421,15 +420,19 @@ export function applyTryTacticToBlock(
...
+    csstree.walk(valueAst, {
+      visit: 'Function',
+      enter(node) {
+        if (node.name === 'anchor') {
```
Comment (jamesnw) [inline src/fallback.ts:427]:
```suggestion
        if (isAnchorFunction(node)) {
```
While `isAnchorFunction` will recheck that it's a function despite the fact that we already know it's a function based on `visit: 'Function'`, I'd prefer to keep using the shared util.

**Review by jamesnw — APPROVED** [review]
Can you also update the Limitations section in the README.md to reflect this improvement? 

Thanks for doing this!

**Pull request merged**